alicloud/table_alicloud_compute_region.go

```diff
@@ -0,0 +1,118 @@
+package alicloud
+
+import (
+	"context"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/ecs"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAlicloudComputeRegion(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "alicloud_compute_region",
+		Description: "",
+		List: &plugin.ListConfig{
+			Hydrate: listComputeRegions,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: []*plugin.Column{
+			// Top columns
+			{
+				Name:        "region",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("RegionId"),
+				Description: ColumnDescriptionRegion,
+			},
+
+			{
+				Name:        "local_name",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LocalName"),
```

Comment:
We are already defined default transform as `transform.FromCamel()`.
No need to use `transform.FromField()`, if it matches the column_name